### ISSUE TITLE
Enforce 0 Findbugs Warnings on Build

### ISSUE THREAD

**Issue by vjkoskela** [ArpNetworking/metrics-portal#46]
Blocked by: https://github.com/sbt/findbugs4sbt/issues/13

**Comment by BrandonArp** [comment]
The find bugs plugin was forked and it looks like the fork has the feature. Let's use the fork to complete this task.

**Comment by vjkoskela** [comment]
Just to be clear you are referring to this fork:

https://github.com/AlejandroRivera/findbugs4sbt

(There seems to be more than one)

**Comment by BrandonArp** [comment]
That is not the one I was referring to. This one was what I was referring to: https://github.com/JosephEarl/sbt-findbugs

**Comment by vjkoskela** [comment]
This is a duplicate of #30 which is now fixed. Closing.
